crosswalks/Java (Maven).csv

```diff
@@ -1,30 +1,30 @@
 Property,Java (Maven)
-codeRepository,scm / repositories
+codeRepository,scm

```

Comment:
Should be `scm/url` according to https://maven.apache.org/pom.html#SCM

Reply:
How about `scm / url` which should be the landing page to the code repository instead of the clone / checkout url

Reply:
Hmm, https://schema.org/codeRepository is ambiguous about which it should be. However "http://127.0.0.1/websvn/my-project" looks way more usable than "scm:svn:http://127.0.0.1/svn/my-project" (which seems to be a maven-ism) so let's go for `scm / url`